app/assets/javascripts/select-kit/addon/components/timezone-input.js

```diff
@@ -4,22 +4,27 @@ import { computed } from "@ember/object";
 export default ComboBoxComponent.extend({
   pluginApiIdentifiers: ["timezone-input"],
   classNames: ["timezone-input"],
-  nameProperty: null,
-  valueProperty: null,
 
   selectKitOptions: {
     filterable: true,
     allowAny: false,
   },
 
+  nameProperty: computed(function () {
```

Comment:
Doesn't have to be done in this PR but a general note: I think all four of these could be done with native getters (instead of 3 `computed()` and one function)